app/frontend/src/main.js

```diff
@@ -112,7 +112,7 @@ async function loadConfig() {
  */
 function loadKeycloak(config) {
   Vue.use(VueKeycloakJs, {
-    init: { onLoad: 'check-sso' },
+    init: { onLoad: 'check-sso'},
```

Comment:
Keep an eye out when you alter this section, so it doesn't add this as a modified file in your commit.